types/hammerjs/index.d.ts

```diff
@@ -212,6 +212,9 @@ declare class HammerInput
 
   /** Reference to the srcEvent.preventDefault() method. Only for experts! */
   preventDefault:Function;
+    
+  /** Additional event(panleft, panright, pinchin, pinchout...) */
+  additionalEvent: string;
```

Comment:
Looks like this property may not be present, so it should be `additionalEvent?: string`.